### DISCUSSION TITLE
Where can I find conversation logs ?

### DISCUSSION THREAD

**Discussion by ltrujilloEquancy** [Pythagora-io/gpt-pilot#985] [General]
Hello, 

I'm looking for a way to keep track of the conversation with gpt pilot. Is there a file that contains the project prompts ? 

Thank you in advance.

**Comment by texorama** [comment]
I assume that this kind of work goes into the development of the (paid) pythagora app?
Anyway, I found just checking out the pythagora.db mysqlite database with is quite helpful, e.g. to know where to "rewind" when you're gone astray.
